.github/workflows/master.yml

```diff
@@ -145,6 +145,16 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      # https://github.com/docker/build-push-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build Kyuubi Docker Image
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: docker/Dockerfile
+          load: true
+          tags: apache/kyuubi:latest
```

Comment:
add `-pl :kyuubi-common_2.12,:kubernetes-integration-tests_2.12` to 
L174 `./build/mvn clean test -Pkubernetes -Dtest=none -DwildcardSuites=org.apache.kyuubi.kubernetes.test`?

Reply:
Done.